corehq/apps/callcenter/management/commands/rebuild_call_center_datasources.py

```diff
@@ -13,7 +13,6 @@
 
 
 class Command(BaseCommand):
-    args = '<domain domain ...>'
```

Comment:
this does work because the handle function doesn't specify *args:
```
def handle(self, domains, **options):
```
So the kwarg `domains` will get pulled out of `options` and supplied as a positional arg.